include/fiddle/postprocess/fiber_network.h

```diff
@@ -0,0 +1,64 @@
+#ifndef included_fiddle_postprocess_fiber_network_h
+#define included_fiddle_postprocess_fiber_network_h
+
+#include <deal.II/distributed/shared_tria.h>
+#include <deal.II/dofs/dof_handler.h>
+#include <deal.II/grid/tria.h>
+
+namespace fdl
+{
+  using namespace dealii;
+    
+    template <int dim, int spacedim = dim>
+    class FiberNetwork
+    {
+        /*
+        stores several cell-centered fiber network data
+        */
+    public:
+
+        FiberNetwork(const Triangulation<dim, spacedim> &tria,
+                    const std::vector<std::vector<Tensor<1, spacedim>>> &fibers)
+                    : tria(&tria)  
+                    {
+                        // initialize local_processor_min_cell_index to maximum value it could be set to
+                        local_processor_min_cell_index = std::numeric_limits<types::global_cell_index>::max();
+
+                        for (const auto &cell : tria.active_cell_iterators())
+                        {
+                            local_processor_min_cell_index = std::min(local_processor_min_cell_index, cell->active_cell_index());
+                        }
+
+                        //number of cells in the mesh
+                        auto n_table_rows = tria.n_active_cells(); // = fibers_vec.size()

```

Comment:
we can just enforce that this is true instead of leaving a comment
```suggestion
                        auto n_table_rows = tria.n_active_cells();
#ifdef DEBUG
                        for (const auto &v : fibers)
                          AssertDimension(n_table_rows, v.size());
#endif
```